server/src/test/java/org/apache/druid/server/coordinator/DruidCoordinatorBalancerTest.java

```diff
@@ -195,6 +201,186 @@ public void testMoveToEmptyServerBalancer()
     Assert.assertEquals(2, params.getCoordinatorStats().getTieredStat("movedCount", "normal"));
   }
 
+  /**
+   * Server 1 has 2 segments.
+   * Server 2 (maintenance) has 2 segments.
+   * Server 3 is empty.
+   * Maintenance has priority 7.
+   * Max segments to move is 3.
+   * 2 (of 2) segments should be moved from Server 2 and 1 (of 2) from Server 1.
+   */
+  @Test
+  public void testMoveMaintenancePriority()
+  {
+    mockDruidServer(druidServer1, "1", "normal", 30L, 100L, ImmutableMap.of(segment1.getIdentifier(), segment1, segment2.getIdentifier(), segment2));
```

Comment:
Line longer than 120 cols. Also below in this PR

Reply:
:white_check_mark: